lexer/gen/golang/acttab.go

```diff
@@ -65,6 +66,8 @@ type action struct {
 }
 
 const actionTableSrc = `
+// generated by gocc; DO NOT EDIT.
```

Comment:
My friend and I were discussing this, and while the output would be a huge win, it may be difficult to get the BNF source name to the code generation parts without changing the API of Gocc too much. Still, it is internal API as far as I'm aware, so this may still be a wise decision.